Scripts/Diagnostics/Azure_DataFactory_DP_Avoid_Plaintext_Secrets.ps1

```diff
@@ -0,0 +1,282 @@
+$debug = $true
+
+function Get-DataFactoryV2() {
+  <#
+    .SYNOPSIS
+    This command shows the data factory.
+    .DESCRIPTION
+    This command shows the data factory.
+    .PARAMETER SubscriptionId
+    The Azure subscription ID containing the Data Factory.
+    .PARAMETER ResourceGroupName
+    The Resource Group name containing the Data Factory.
+    .PARAMETER DataFactoryName
+    The Data Factory name.
+    .INPUTS
+    None
+    .OUTPUTS
+    Text with the Data Factory name and its Tag names/values.
+    .EXAMPLE
+    PS> Get-DataFactoryV2 -SubscriptionId "00000000-xxxx-0000-xxxx-000000000000" -ResourceGroupName "MyResourceGroupName" -DataFactoryName "MyDataFactoryName"
+    .LINK
+    None
+  #>
+
+  [CmdletBinding()]
+  param (
+    [Parameter(Mandatory = $true)]
+    [string]
+    $SubscriptionId,
+    [Parameter(Mandatory = $true)]
+    [string]
+    $ResourceGroupName,
+    [Parameter(Mandatory = $true)]
+    [string]
+    $DataFactoryName
+  )
+
+  $profile = Set-AzContext -Subscription $SubscriptionId
+
+  $factory = Get-AzDataFactoryV2 -ResourceGroupName $ResourceGroupName -Name $DataFactoryName
+
+  Write-Debug -Debug:$debug -Message ("Data Factory: " + $factory.DataFactoryName)
+
+  Write-Debug -Debug:$debug -Message "Tags:"
+  if ($factory.Tags) {
+    foreach ( $tag in $factory.Tags.GetEnumerator() ) {
```

Comment:
This can potentially result in secret copying,

Reply:
Let's add some NOTE for that, to may be clear the console using cls command.

Reply:
Removed file